package/kernel/linux/modules/crypto.mk

```diff
@@ -11,6 +11,18 @@ CRYPTO_MODULES = \
 	ALGAPI2=crypto_algapi \
 	BLKCIPHER2=crypto_blkcipher
 
+CRYPTO_MIPS32=ath25 ath79 bcm47xx bcm63xx lantiq/generic malta/be malta/le \
+	      pistachio ramips realtek
+CRYPTO_MIPS64=malta/be64 malta/le64 octeon
+CRYPTO_ARM32_NEON=armvirt/32 bcm27xx/bcm2709 imx6 ipq40xx ipq806x \
+		  layerscape/armv7 mediatek/mt7623 mvebu/cortexa9 \
+		  omap sunxi/generic zynq
+CRYPTO_ARM32=at91 bcm27xx/generic bcm53xx gemini kirkwood mediatek/generic \
+	     mediatek/mt7629 mxs oxnas tegra $(CRYPTO_ARM32_NEON)
+CRYPTO_ARM64=armvirt/64 bcm27xx/bcm2710 bcm27xx/bcm2711 bcm4908 ipq807x \
```

Comment:
If you're having a separate ARM32_NEON category, perhaps you should also name this one ARM64_NEON, so that it's consistent with what it represents.

Reply:
Reworked this - please take a look.